server/test/api/purchase.js

```diff
@@ -0,0 +1,149 @@
+/* global describe, it, beforeEach */
+
+var chai = require('chai');
+var expect = chai.expect;
+var helpers = require('./helpers');
+var uuid    = require('node-uuid');
+helpers.configure(chai);
+
+
+/**
+* The /projects API endpoint
+*
+* This test suite implements full CRUD on the /projects HTTP API endpoint.
+*/
+
+var PURCHASE_KEY = ['uuid', 'reference', 'cost', 'discount', 'purchase_date', 'paid', 'text', 'name', 'prenom', 'first',
+  'last', 'creditor_uuid', 'timestamp', 'note', 'paid_uuid', 'confirmed', 'closed', 'is_direct', 'is_donation', 'emitter_id',
+  'is_authorized', 'is_validate', 'confirmed_by', 'is_integration', 'purchaser_id', 'receiver_id'];  
+
+describe('The /Purchase Order API endpoint', function () {
+  var agent = chai.request.agent(helpers.baseUrl);
+
+
+  // purchase we will add during this test suite.
+  var purchase_order = {
+      uuid          : uuid(),
```

Comment:
Please use `uuid.v4()` instead of `uuid()`.  This method is deprecated and will be removed in future versions of `node-uuid`.